### PR TITLE
update edit fix

### PR DESCRIPTION
**Pull request by sethpollack**
per #881

### CODE REVIEW

File: pkg/types/kustomization.go [hunk 1]
```diff
@@ -171,6 +172,50 @@ func DealWithDeprecatedFields(data []byte) []byte {
 	return data
 }
 
+func (k *Kustomization) DealWithLegacyKVGenerators() []string {
```
Comment (monopole) [inline pkg/types/kustomization.go]: this is sufficiently complex that it should have a regression test in kustomization_test.go (which doesn't exist yet).

**Comment by monopole** [comment]
this has all moved now, thanks Seth!

**Comment by sethpollack** [comment]
👍

**Pull request closed without merging**